tests/xharness/AppRunner.cs

```diff
@@ -12,6 +12,7 @@
 using System.Xml.Xsl;
 using Xamarin;
 using Xamarin.Utils;
+using xharness.Logging;
```

Comment:
I know the original one did not but it should start with a capital letter  :)

Reply:
easy fix